integrations/api_feed_user_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFeed(t *testing.T) {
+	t.Run("User", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("User feed", func(t *testing.T) {
```

Reply:
It get's displayed as `TestFeed/User/Atom`. Do you think the additional `feed` is neccesary? When org support is added, there should be `TestFeed/Org/Atom`.

Reply:
Nvm then, test's name is being displayed.